docs/features/sharepoint.md

```diff
@@ -18,24 +18,18 @@ Your SharePoint files will be stored in the blob storage created by Information
 
 ### New Local Dev Variables
 
-#### SharePoint Sites
-
-This is a comma delimited list of SharePoint site(s) you wish to ingest. **It shares a one-to-one relationship with the SharePoint folders variable.** If you have one SharePoint site, you need one SharePoint folder and vice versa. You may list the same site twice if you wish to more granular control over what is being crawled in SharePoint.
-
-#### SharePoint Folders
-
-This is a comma delimited list of SharePoint folder(s) you wish to be your entry point for the ingestion. **It shares a one-to-one relationship with the SharePoint sites variable.** If you have one SharePoint folder, you need one SharePoint site and vice versa. It will crawl this folder and then automatically crawl any folders beneath it. The SharePoint site root is typically /Shared Documents
+#### SHAREPOINT_TO_SYNC
+This is a single quoted JSON Array of Objects within the env file. With the keys "url" and "folder" for each sharepoint site to ingest.
+```JSON
+export SHAREPOINT_TO_SYNC='[
+    { "url": "https://yoursharepoint.com", "folder": "Shared Documents"},
```

Comment:
In the MD example in deployment.md the folder is represented as "/Shared Documents". Is the forward slash required or not? Please indicate and make consistent across the documentation.

Reply:
It is required. Good catch. Fixed.